src/contracts/interfaces/IBLSRegistryCoordinatorWithIndices.sol

```diff
@@ -50,4 +52,18 @@ interface IBLSRegistryCoordinatorWithIndices is ISignatureUtils, IRegistryCoordi
     function blsPubkeyRegistry() external view returns (IBLSPubkeyRegistry);
     /// @notice the Index Registry contract that will keep track of operators' indexes
     function indexRegistry() external view returns (IIndexRegistry);
+
+    /**
+     * @notice Ejects the provided operator from the provided quorums from the AVS
+     * @param operator is the operator to eject
+     * @param quorumNumbers are the quorum numbers to eject the operator from
+     * @param pubkey is the BLS public key of the operator
+     * @param operatorIdsToSwap is the list of the operator ids tho swap the index of the operator with in each
```

Comment:
incomplete sentence(?) -- feels like this just trails off without completing the thought
also typo: `tho` => `to`

Reply:
c1740bc8 this work with ya?